resources/install.sh

```diff
@@ -20,6 +23,16 @@ password=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | tr -
 sed -i s/MTIxMzQyNDMyZHdkY2RldmdyZWc=/$password/g /var/lib/gravity/resources/secrets.yaml
 
 /opt/bin/kubectl apply -f /var/lib/gravity/resources/namespace.yaml
+cat <<EOF | /opt/bin/kubectl create -f -
```

Comment:
What about upgrades? I didn't see this config map created in upgrade hook - new pods will try to mount it and fail?

Reply:
Also, do we even need this config map? Is it possible to use downward API maybe to mount the security context user into containers maybe? https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/